src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -107,6 +229,39 @@ InvokerFactory MakeImplGemmDynamicDataInvokerFactory(const ConvolutionContext& c
             };
         };
     }
+    else if(ctx.direction.IsBackwardData())
+    {
+        return [ctx](const std::vector<Kernel>& kernels) {
```

Comment:
Since you are not modifying ctx inside the lambda function, you could pass it as reference i.e return [&ctx]

Reply:
No, because it would get destroyed at the run stage or even earlier.

Reply:
seems no change needed?

Reply:
@DrizztDoUrden I see now why you had to pass it by value.